library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -80,21 +80,12 @@ public int getSpanCount() {
     }
 
     /**
-     * Updates the adapter with a new list that will be diffed on a background thread
-     * and displayed once diff results are calculated.
-     *
-     * NOTE: This update method is NOT compatible with partial updates (change notifications
-     * driven by individual groups and items).  If you update using this method, all partial
-     * updates will no longer work and you must use this method to update exclusively.
-     * <br/> <br/>
-     * If you want to receive a callback once the update is complete call the
-     * {@link #updateAsync(List, OnAsyncUpdateListener)} version
-     *
-     * @param newGroups List of {@link Group}
+     * Overload for {@link GroupAdapter#updateAsync(List, OnAsyncUpdateListener, boolean)}
```

Comment:
Please further specify that this defaults `detectMoves` to true.

Reply:
I did not explicitly specify that since `true` is the default behavior of `DiffUtil`. The docs of `DiffUtil` itself also do not specify this. Let me know if you would still want it added

Reply:
I think yes - it's good to know for library users